operations/customize_disk_hardware.md

```diff
@@ -0,0 +1,32 @@
+# Customize PCIe Hardware
+
+This page will assist an admin with changing the kernel parameters for NCNs that have extra disks.
+
+> **`NOTE:`** If a system's hardware is Plan of Record (PoR) then this page is not needed.
+
+For any procedure below, it is assumed that the extra disks are going to be utilized. If they are undesired, then the only action item todo is to yank/remove/pull the disks from the NCN.
+
+## Procedure
+
+### Masters & Workers
+
+Add the kernel parameter `metal.disks=X` where `X` is the number of extra disks on the NCN. This parameter may be added to the PIT's boot scripts or to BSS
```

Comment:
- The second sentence is missing a period
- Spell out Boot Script Service (BSS) for first usage in this procedure